### DISCUSSION TITLE
Waiting for a new release

### DISCUSSION THREAD

**Discussion by pfeigl** [coverlet-coverage/coverlet#1716] [General]
Hi,

we are desperatly waiting for a new release. We are stuck on two problems which actively block generating our coverage for a WPF based solution.

Boths problems have been fixed and merged already quite some time ago. Specifically, it's these two:
* https://github.com/coverlet-coverage/coverlet/pull/1654
* https://github.com/coverlet-coverage/coverlet/pull/1705

Is there any release plan for a new version - atleast a hotfix release perhaps?

Thanks for your help,
Philipp

**Comment by Bertk** [comment]
<img width="739" height="511" alt="image" src="https://github.com/user-attachments/assets/cbe33474-7288-45f1-ba24-3e416abb4a42" />
